--- at-spi2-core-2.26.2/bus/Makefile.am.0000~	2017-11-25 16:11:07.745018268 +0100
+++ at-spi2-core-2.26.2/bus/Makefile.am	2017-11-25 16:11:25.994045009 +0100
@@ -29,7 +29,7 @@ org.a11y.Bus.service: org.a11y.Bus.servi
 EXTRA_DIST += org.a11y.Bus.service.in
 CLEANFILES += org.a11y.Bus.service
 
-systemd_userdir = $(prefix)/lib/systemd/user
+systemd_userdir = /lib/systemd/user
 systemd_user_DATA = at-spi-dbus-bus.service
 at-spi-dbus-bus.service: at-spi-dbus-bus.service.in Makefile
 	$(AM_V_GEN) $(SED) -e $(substitutions) $< > $@.tmp && mv $@.tmp $@
